django_jsonform/contrib/dataclasses/utils.py

```diff
@@ -0,0 +1,24 @@
+from __future__ import annotations
+
+import dataclasses
+from typing import Type
+
+from dataclasses_jsonschema import JsonSchemaMixin
```

Comment:
I don't think we should add this as a dependency to this project.

Need to research how to make this `contrib` module optional.

Maybe removing the `contrib/__init__,py` would work?